tests/integration/linux_bridge_test.py

```diff
@@ -76,6 +77,16 @@ def bridge0_with_port0(port0_up):
         yield deepcopy(desired_state)
 
 
+@pytest.fixture
+def port0_vlan101(port0_up):
+    vlan_id = '101'
```

Comment:
This should be an integer and not a string.
You need to convert it to a string when you concatenate the name later.